### PR TITLE
Several minor revisions

### PR DESCRIPTION
**Pull request by rohanthewiz**
The idea here is to clean up some grammatical issues, but also to tighten up the wording a bit.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (rohanthewiz) [inline .gitignore:1]: It is good to add a `.gitignore` file to any serious project.
Here I am ignoring JetBrains IDE configs -- those are personal and don't need to be shared and clutter up the project.
This also makes it easy to do `git add -A` to quickly stage changes without having to hunt and pick.

File: Host a Static Website with Hugo and NGINX.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Throughout this guide commands will be given with example variables to provide g
 | username                 | Replace with Linode login user name                   |
 
 
-Begin by completing the Getting Started guide and Securing Your Server guide if you are getting started with Linode for the first time.
+Begin by completing the following guides if you are getting started with Linode for the first time.
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:48]: Listing the guides ahead of time without links, makes for a less than ideal user experience. So just tell 'em it's coming.

File: Host a Static Website with Hugo and NGINX.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ Configuration steps for the Linode side can be found at [DNS Manager Overview](/
 
 ### Set Linode Name Servers on Web Hosting
 
-The typical scenario is that you purchased and registered a domain name through one of the popular shared web hosting provider. Now you need to have the domain name point to your Linode server. Navigate to the Control Panel on your shared web hosting.
+The typical scenario is that you purchased and registered a domain name through one of the popular providers such as Bluehost, Godaddy, or Fatcow. Now you need to have the domain point to your Linode server. Navigate to the Control Panel on your shared web hosting.
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:72]: Also last sentence: "Navigate to your providers Domain control panel." You may have to make that change after the merge of this pull request, if you choose to merge.

File: Host a Static Website with Hugo and NGINX.md (hunk 2 above)
Comment (jaminyah) [reply Host a Static Website with Hugo and NGINX.md:48]: This guide is linked to Linode's internal docs.

File: Host a Static Website with Hugo and NGINX.md [hunk 4]
```diff
@@ -108,8 +108,7 @@ The following name server entries should be made:
 * ns4.linode.com
 * ns5.linode.com
 
-It is best of follow the instructions provided for your specific scenario and registrar. The scenario discussed above applies to previously purchased through one of the common registrars such as Bluehost, Godaddy, or Fatcow.
-
+The above is given as an example. Follow the instructions for your specific domain provider.
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:111]: Again, the article is long enough. Keep things short and sweet.

File: Host a Static Website with Hugo and NGINX.md [hunk 5]
```diff
@@ -119,7 +118,7 @@ One to the things you may want to do is add a subdomain to your domain and have
   <img src="/images/AAAA.jpg" alt="AAAA records" /> 
 </p>
 
-In the section for A/AAAA Records select the link 'Add a new A record'.  For Hostname add `http://blog.example.com`. In the IP Address field enter the IP address of your Linode server. 
+In the section for A/AAAA Records select the link "Add a new A record".  For Hostname add `http://blog.example.com`. In the IP Address field enter the IP address of your Linode server.
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:121]: The name of some things can be quoted with double quotes. Consistency is a good also.

File: Host a Static Website with Hugo and NGINX.md [hunk 6]
```diff
@@ -141,7 +140,7 @@ Enter the `username` that you use to log into your Linode server. This user must
 Enter the `IPAddress` of your Linode server.
 {{< /note>}}
 
-Further clarification on logging in with SSH can be found in the [Getting started](/docs/getting-started) guide in the section titled *Log In for the First Time*. Create a directory named hugo and change into this directory. 
+Further clarification on logging in with SSH can be found in the [Getting started](/docs/getting-started) guide in the section titled *Log In for the First Time*. Create a directory named "hugo" and change into this directory.
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:143]: Also perhaps "Log In for the First Time" should be in double quotes for consistency.

File: Host a Static Website with Hugo and NGINX.md [hunk 7]
```diff
@@ -155,7 +154,7 @@ Use the `wget` command to get the latest Hugo release files. Latest releases are
     wget https://github.com/gohugoio/hugo/releases/download/v0.31.1/hugo_0.31.1_Linux-64bit.deb
 
 {{< note >}}
-To get the link to the release you wish to obtain right click on the release link and select Copy Link. Paste in the Terminal window.
+To get the link to the release you wish to obtain right click on the release link and select "Copy Link". Paste in the Terminal window.
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:157]: Ah, "To get the link of the release..."

File: Host a Static Website with Hugo and NGINX.md [hunk 8]
```diff
@@ -326,7 +325,7 @@ Press the **Escape** key, then press **:wq** to end editing of the config file.
 {{< /note >}}
 
 ### Vim Editor Basics
-
+{{Rohan: This section really doesn't belong in the scope of this article. Replace with an external link to a similar VI quick-start.}}
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:328]: Yeah, the article is already too long, so this section is out of scope. Provide an external link in the "Configuring Nginx" section and ditch this section.

File: Host a Static Website with Hugo and NGINX.md [hunk 9]
```diff
@@ -451,16 +450,16 @@ It is very possible that you may attempt to stop or reload the Nginx server and
...
-To start, stop, or reload Nginx it is necessary to use the `sudo` prefix along with the command to stop, reload or start Nginx. That is, `sudo systemctl stop nginx`, `sudo systemctl reload nginx`, `sudo systemctl start nginx`.
+To start, stop, or reload Nginx it is necessary to prefix the command with `sudo`. For example: `sudo systemctl reload nginx`.
 
 ### Hugo Server did not Shutdown Using **CTRL+C**
 
-There are times when shutting down the server with **CTRL+C** doesn't do the trick. This may be apparent the next time we attempt to start-up the Hugo server. For example, the Terminal console may display something to the effect, "Web Server is available at `http://localhost:12345`". 
+There are times when shutting down the server with **CTRL+C** doesn't do the trick. This may be apparent the next time we attempt to start-up the Hugo server. For example, the Terminal console may display something similar to `web server is available at http://localhost:12345`. 
 
-Here is the scenario that can result in this condition:
+Here is a scenario that can result in this condition:
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:459]: So, rarely do we want to say this is the **only** way to do something. Leave the possibilities open. It is "a" way.

File: Host a Static Website with Hugo and NGINX.md [hunk 10]
```diff
@@ -477,7 +476,7 @@ Terminal output should be similar to:
 
     [1]-  Terminated     hugo server -t material-design -D --baseUrl="http://blog.example.com" --appendPort=false
 
-However, stopping the process on port 1313 is not enough since a new Hugo process was fired up on port 12345 because port 1313 was not available. In this situation running command `ps aux | grep hugo` should show that there are more than one Hugo servers running. Each is running with a different Process ID.
+However, stopping the process on port 1313 is not enough since a new Hugo process was fired up on port 12345 when port 1313 was not available. In this situation running the command `ps aux | grep hugo` should show that there is more than `hugo server` running. Note the different process ids.
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:479]: Oops: "...should show that there is more than _one_ `hugo server` ...

File: Host a Static Website with Hugo and NGINX.md [hunk 11]
```diff
@@ -496,29 +495,23 @@ A better approach is to use the pkill command.
 
     pkill hugo
 
-Then show the list of hugo processes.
-
-    ps aux | grep hugo
-
-Which should return output:
+Now `ps aux | grep hugo` should show no hugo servers running:
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:498]: Make it concise.

File: Host a Static Website with Hugo and NGINX.md [hunk 12]
```diff
@@ -496,29 +495,23 @@ A better approach is to use the pkill command.
...
 
     username 16628  0.0  0.1  14224  1020 pts/1    S+   15:29   0:00 grep --color=auto hugo
 
 ### Web Browser Returns Bad Gateway
 
-Bad Gateway can be returned by the web browser if the Hugo server is not running or is not running at port 1313. 
+"Bad Gateway" can be returned by the web browser if the Hugo server is not running or is not running at port 1313.
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:504]: One more thing: "server is not running _on_ port 1313..."

File: Host a Static Website with Hugo and NGINX.md [hunk 13]
```diff
@@ -496,29 +495,23 @@ A better approach is to use the pkill command.
...
+"Bad Gateway" can be returned by the web browser if the Hugo server is not running or is not running at port 1313. 
 
 <p align="center">
   <img src="/images/BadGateway.jpg" alt="Bad Gateway web page" /> 
 </p>
 
-One way to find out more about status of the Hugo server is to issue command:
-
-    ps aux | grep hugo
+In this case you can use `ps aux | grep hugo` to verify the server is not running.
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:510]: No need to repeat the info already given just above.

File: Host a Static Website with Hugo and NGINX.md (hunk 7 above)
Comment (jaminyah) [reply Host a Static Website with Hugo and NGINX.md:157]: I saw that comment. Issue is I had a hard time finding the link to the release even with the releases listed. If you click on a release it automatically downloads. Now you are stuck with deciding if using FTP is a better option since it's downloaded already. Even copy + paste into most applications hides the actual URL by making it a clickable link. So getting the URL may not be as obvious as it appears.

File: Host a Static Website with Hugo and NGINX.md (hunk 8 above)
Comment (jaminyah) [reply Host a Static Website with Hugo and NGINX.md:328]: Interestingly, I agree. The issue is the Vim site doesn't even have a section with Vim Editor basics. I'll see if there are any good reputable options. I like the idea of a link. Issue is a reputable site. Should be Vim site but no such content.

File: Host a Static Website with Hugo and NGINX.md [hunk 14]
```diff
@@ -496,29 +495,23 @@ A better approach is to use the pkill command.
...
 
-If Hugo is stopped the command returns an output similar to:
+Restart the server, then check for any errors:
 
-    username 15873  0.0  0.1  14224  1024 pts/2    S+   12:29   0:00 grep --color=auto hugo
+    hugo server -t YOUR-TEMPLATE-NAME -D --baseUrl="http://blog.YOUR-DOMAIN-NAME" --appendPort=false &
```
Comment (rohanthewiz) [inline Host a Static Website with Hugo and NGINX.md:514]: Hey, while you are testing, verify first that this is the command you want here, but also with the "&", verify that the server keeps running after logging out. I get the feeling it wont.
I think it is better to use the nohup (no hangup command). Here is an example for a nodejs server
`nohup nodejs server.js --be_ip 10.128.0.2 --fe_ip 10.128.0.3 &`
so `nohup server --options ... &`

**Review by rohanthewiz — COMMENTED** [review]
Yeah, fantastic job. I know this was a ton of work. So adding some minor tweaks, particularly in the area of conciseness. You may want to go over these first and make note of a couple after-the-fact changes -- there are a couple of them. Merge in the Pull request, then add those changes in a later commit. Just remember to do a `git pull` before making those final changes.

File: Host a Static Website with Hugo and NGINX.md (hunk 14 above)
Comment (jaminyah) [reply Host a Static Website with Hugo and NGINX.md:514]: I explored this `&` issue some to see what was the simplest with the least amount of side effect and effort. So, yes the Hugo server keeps running and has been running for several weeks. Even after shutdown and restart on various occasions.

File: Host a Static Website with Hugo and NGINX.md (hunk 13 above)
Comment (jaminyah) [reply Host a Static Website with Hugo and NGINX.md:510]: Agreed.

File: Host a Static Website with Hugo and NGINX.md (hunk 2 above)
Comment (rohanthewiz) [reply Host a Static Website with Hugo and NGINX.md:48]: Yes, but no need to list them twice. Just say the links are below.

File: Host a Static Website with Hugo and NGINX.md (hunk 12 above)
Comment (jaminyah) [reply Host a Static Website with Hugo and NGINX.md:504]: I am using `at port 1313` numerous places throughout the doc. So I'll correct this issue.

File: Host a Static Website with Hugo and NGINX.md (hunk 9 above)
Comment (jaminyah) [reply Host a Static Website with Hugo and NGINX.md:459]: Agree.

File: Host a Static Website with Hugo and NGINX.md (hunk 8 above)
Comment (rohanthewiz) [reply Host a Static Website with Hugo and NGINX.md:328]: What would be nice is to (later possibly) point this to your own blog of VIM Basics.

File: Host a Static Website with Hugo and NGINX.md (hunk 5 above)
Comment (jaminyah) [reply Host a Static Website with Hugo and NGINX.md:121]: Agree.

File: Host a Static Website with Hugo and NGINX.md (hunk 14 above)
Comment (rohanthewiz) [reply Host a Static Website with Hugo and NGINX.md:514]: There are likely doing something like a hibernate then. NP then as long as it works on Linode.
Btw, I got that nohup nodejs server line from a Google Cloud Kubernetes tutorial for running a nodejs server inside a docker container.

Comment (rohanthewiz) [reply Host a Static Website with Hugo and NGINX.md:514]: For the purposes of this article though, it is probably best to stick with what you have.

File: Host a Static Website with Hugo and NGINX.md (hunk 6 above)
Comment (jaminyah) [reply Host a Static Website with Hugo and NGINX.md:143]: This statement migrated into the wrong place during the updates. It needed to be in the "Installing Hugo" section.

File: .gitignore (hunk 1 above)
Comment (jaminyah) [reply .gitignore:1]: Done.
